tests/Interface/core/ImageContainerTest.cpp

```diff
@@ -332,6 +469,52 @@ int main() {
   const Img<float, 2> testBracketOperator6(grayimage_png);
   // CHECK: 240.0
   fprintf(stderr, "%f\n", testBracketOperator6[15]);
-  
+
+  //===--------------------------------------------------------------------===//
```

Comment:
These tests are testing the same thing which was tested by above tests? If not, please add a comment explaining that.

Reply:
yes , above tests checks only on single images , here i have batched two images into a blob that is the only difference 

and i have added few required  segments in ImageContainer.h for batched images